### PR TITLE
feat: checkbox and minor styling fixes

### PR DESCRIPTION
**Pull request by lessp**
Add Checkbox based on Desmond's new design.

Also adds minor fixes to CTAButton & FormInput

### CODE REVIEW

File: lib/FormElements/Checkbox.tsx [hunk 1]
```diff
@@ -28,18 +37,66 @@ const CheckboxInput = styled.input`
...
+`
+
+const CheckboxText = styled.p`
+  align-self: flex-start;
+  color: ${({ theme }) => theme.colors.shipGrey};
+  cursor: pointer;
+  flex: 1;
+  font-family: ${({ theme }) => theme.typography.medium};
+  font-size: 14px;
+  font-size: 12px;
```
Comment (AElmoznino) [inline lib/FormElements/Checkbox.tsx]: Again, 2 font-sizes set :)

File: lib/FormElements/Checkbox.tsx [hunk 2]
```diff
@@ -1,25 +1,34 @@
 import * as React from 'react'
 import styled, { injectGlobal } from '../styles/themed-components'
 
+const checkedIcon = `data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMiIgaGVpZ2h0PSIxMiIgdmlld0JveD0iMCAwIDEyIDEyIj4gICAgPHBhdGggZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIiBzdHJva2U9IiNGRkYiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSIyIiBkPSJNMSA2bDQgNSA2LTEwIi8+PC9zdmc+`
+
 export interface CheckboxProps {
   'data-test'?: string
+  checked?: boolean
```
Comment (AElmoznino) [inline lib/FormElements/Checkbox.tsx]: Seems like checked is always required (at least it defaults so `false` further down, if that's the case, remove the `?`-mark

File: lib/FormElements/Checkbox.tsx [hunk 3]
```diff
@@ -28,18 +37,66 @@ const CheckboxInput = styled.input`
...
 `
 
 const CheckboxLabel = styled.label`
-  color: ${({ theme }) => theme.colors.mountainMist};
+  color: ${({ theme }) => theme.colors.shipGrey};
   cursor: pointer;
   flex: 1;
+  font-family: ${({ theme }) => theme.typography.medium};
   font-size: 14px;
+  font-size: 16px;
```
[the remark this answers is not in the document]
Comment (lessp) [reply lib/FormElements/Checkbox.tsx]: It's a mixture 👼😄

File: lib/FormElements/Checkbox.tsx [hunk 4]
```diff
@@ -28,18 +37,66 @@ const CheckboxInput = styled.input`
...
   }
 `
 
 const CheckboxLabel = styled.label`
-  color: ${({ theme }) => theme.colors.mountainMist};
+  color: ${({ theme }) => theme.colors.shipGrey};
   cursor: pointer;
   flex: 1;
+  font-family: ${({ theme }) => theme.typography.medium};
   font-size: 14px;
```
Comment (lessp) [inline lib/FormElements/Checkbox.tsx]:
```suggestion
```

File: lib/FormElements/Checkbox.tsx [hunk 5]
```diff
@@ -28,18 +37,66 @@ const CheckboxInput = styled.input`
...
+  }
+`
+
+const CheckboxText = styled.p`
+  align-self: flex-start;
+  color: ${({ theme }) => theme.colors.shipGrey};
+  cursor: pointer;
+  flex: 1;
+  font-family: ${({ theme }) => theme.typography.medium};
+  font-size: 14px;
```
Comment (lessp) [inline lib/FormElements/Checkbox.tsx]:
```suggestion
```

File: lib/FormElements/Checkbox.tsx (hunk 2 above)
Comment (lessp) [reply lib/FormElements/Checkbox.tsx]: I think you still have to pass it a checked-prop if it's required, but it's probably more convenient to pass it `checked={true}` and not have to worry about passing it false as well?

Comment (AElmoznino) [reply lib/FormElements/Checkbox.tsx]: Good question, I'm not sure, you may be right :)

**Pull request merged**